packages/context/src/inject-config.ts

```diff
@@ -159,28 +200,28 @@ function resolveAsViewFromConfig(
   session: ResolutionSession,
 ) {
   assertTargetType(injection, ContextView);
-  const bindingKey = getCurrentBindingKey(session);
+  const bindingKey = getTargetBindingKey(injection, session);
   // Return `undefined` if no current binding is present
   if (!bindingKey) return undefined;
   const view = new ConfigView(
     ctx,
     binding =>
       binding.key === BindingKey.buildKeyForConfig(bindingKey).toString(),
-    injection.metadata.configPath,
+    injection.metadata.propertyPath,
```

Comment:
Should we keep support for `meta.configPath` for backwards compatibility?

```suggestion
    injection.metadata.propertyPath || injection.metadata.configPath,
```

If yes, then please include also a `TODO(semver-major)` comment to remove that compatibility code in the next major version.